### PR TITLE
Migrate to using license feature (JBIDE-20202...
...)

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

File: common/features/org.jboss.tools.common.all.feature/feature.properties [hunk 1]
```diff
@@ -1,9 +1,6 @@
 featureName=JBoss Tools Common All Feature
 featureProvider=JBoss by Red Hat
 
-# "updateSiteName" property - label for the update site
-updateSiteName=JBossTools Update Site
```
Comment (dazarov) [inline common/features/org.jboss.tools.common.all.feature/feature.properties]: How is this related to the license staff? Do we need to delete this in all features?

Comment (nickboldt) [inline common/features/org.jboss.tools.common.all.feature/feature.properties]: Not related. But was updating feature.properties and saw these unreferenced properties so figured I'd remove them too. They're useless clutter.

**Comment by dazarov** [comment]
This PR not passes STEPS TO VERIFY THAT IT WORKED from https://issues.jboss.org/browse/JBIDE-20194.
There is only one common feature in Help → Installation Details
org.jboss.tools.common.jdt.feature from 8.
We need to fix this.
I am working on PR which do both 
1. make feautures be visible in  Installation Details
2. make license work

**Comment by nickboldt** [comment]
What do you mean it doesn't pass? Worked fine at my end.

Help > Install Details for me shows all the foundation & common features, including source features. All have licenses. 

I checked for common, foundation, stacks, and runtime. 

See screenshots here:

https://issues.jboss.org/browse/JBIDE-20202?focusedCommentId=13100368&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13100368

**Comment by alexeykazakov** [comment]
Squished and pushed to master.
